nodebuilder/p2p/flags.go

```diff
@@ -68,12 +69,18 @@ func ParseFlags(
 // and returns either the parsed network or the build's default network
 func ParseNetwork(cmd *cobra.Command) (Network, error) {
 	parsed := cmd.Flag(networkFlag).Value.String()
+	if parsed == "default" {
```

Comment:
Compare against the string behind `DefaultNetwork`

Reply:
You have rightly pointed out. Compare against would be better here.